### PR TITLE
[#155] Nation and community button in dashboard (connects #155)

### PR DESCRIPTION
**Pull request by Rupesh87**

### CODE REVIEW

File: src/app/dashboard/dashboard.component.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            <thead>
+            <tr>
+                <th i18n>Nation Name</th>
+                <th i18n>Admin Name</th>
+                <th i18n>Nation Url</th>
+            </tr>
+            </thead>
+            <tbody>
+            <tr *ngFor="let nations of nation; let i = index">
+                <td>{{ nations.doc.name }}</td>
```
Comment (paulbert) [inline src/app/dashboard/dashboard.component.html]: The name for nations should be `nation_name`.  Mock data is fixed to use `nation_name` in #154

**Comment by paulbert** [comment]
The issue just says this would add buttons to go to nation and community pages.  Do we need the tables?

Also either way we should restrict this role based on the user roles.  The currently logged in user information is stored in the UserService.

File: src/app/dashboard/dashboard.component.html (hunk 1 above)
Comment (Rupesh87) [reply src/app/dashboard/dashboard.component.html]: @paul nation name has been changed to nation_name

**Comment by mappuji** [comment]
Do review with this step please 
https://www.youtube.com/watch?v=SmZgAULwoh4

**Comment by paulbert** [comment]
Closing it since it duplicates #176

**Pull request closed without merging**